Makefile

```diff
@@ -140,8 +140,8 @@ manager: codegen fmt vet ## Build manager binary
 
 .PHONY: manifests
 manifests: ${CONTROLLER_GEN} ## Generate manifests e.g. CRD, RBAC etc.
-	cd pkg/sdk && $(CONTROLLER_GEN) $(CRD_OPTIONS) webhook paths="./..." output:crd:artifacts:config=../../config/crd/bases output:webhook:artifacts:config=../../config/webhook
```

Comment:
I don't think we can remove CRD_OPTIONS, as for example `maxDescLen=0` is limiting our crds in size. I attribute at least some of the test failures to this.

Reply:
Added maxdesclen, others are deprecated.